Fix label being cutoff in Newsletter registration

folektoras opened the pull request:
In this commit the label to Register in the newsletter that exists in the homepage had the text completely cut off on Tablet iOS devices

### Description
To solve this the width property was altered to use the max-content value, and a max-width property is also added. Also some paddings were set to 0 because Safari keeps adding some default padding values.

### Fixed Issues 
1. magento/magento2#21592: Newsletter subscription input box in footer: on iPad (9.7"), in portrait mode, placeholder text is not visible

### Manual testing scenarios
1. Go to the homepage of a clean Magento 2.3 installation on an iOS tablet device 9.7"
2. Check the Newsletter Subscription input field at the footer of the page
3. Make sure the input text is fully visible
4. Now visit the page on other browsers too and make sure that this still fixed

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

folektoras committed:
Fix label being cutoff in Newsletter registration

In this commit the label to Register in the newsletter that exists in the homepage had the text completely cut off on Tablet iOS devices

To solve this the width property was altered to use the max-content value, and a max-width property is also added. Also some paddings were set to 0 because Safari keeps adding some default padding values.

folektoras committed:
Merge pull request #1 from cstergianos/feature/21592-newsletter-subscription-input-box-fix

Fix label being cutoff in Newsletter registration

folektoras committed:
Sort alphabetically the less properties

During deployment an error was thrown that the properties are not sorted alphabetically. This is now solved.

magento-engcom-team commented:
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6147](https://jira.corp.magento.com/browse/ENGCOM-6147) has been created to process this Pull Request
:eight_spoked_asterisk: @dmytro-ch, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

engcom-Alfa commented:
Hi @cstergianos 

Could you also solve this problem and for the "Blank" theme?

**Actual Result:**
![after](https://user-images.githubusercontent.com/51679138/67288692-14323180-f4e6-11e9-9788-86c88877ae20.png)

Thanks!

cstergianos commented:
> Hi @cstergianos
> 
> Could you also solve this problem and for the "Blank" theme?
> 
> **Actual Result:**
> ![after](https://user-images.githubusercontent.com/51679138/67288692-14323180-f4e6-11e9-9788-86c88877ae20.png)
> 
> Thanks!

Ow yeah that makes a lot of sense! Will do as soon as possible :)

folektoras committed:
Apply the same fix for Magento blank

In this commit two changes are introduced. The first one is removing the .lib-css while applying the width style property and the other one is to apply the same changes for Magento Blank theme.

folektoras committed:
Merge pull request #2 from cstergianos/feature/21592-newsletter-subscription-input-box-fix

Apply the same fix for Magento blank

magento-engcom-team commented:
Hi @dmytro-ch, thank you for the review.
[ENGCOM-6147](https://jira.corp.magento.com/browse/ENGCOM-6147) has been created to process this Pull Request

engcom-Alfa commented:
:heavy_check_mark: QA Passed
